test/dynamo/test_misc.py

```diff
@@ -627,66 +627,6 @@ def h(x):
             cleanup_op("mylib::bar3")
             del lib
 
-    def test_auto_functionalize_can_with_default(self):
```

Comment:
nit: code movement would have been better in a separate PR on the stack so that this one doesn't look like +1200 -500 :P. But it's OK for this PR, I can still tell what's going on.

Reply:
done